models/eprop_synapse.h

```diff
@@ -462,22 +461,15 @@ eprop_synapse< targetidentifierT >::send( Event& e, size_t thread, const EpropSy
   assert( target );
 
   const long t_spike = e.get_stamp().get_steps();
-  long t_begin;
 
-  if ( is_first_spike_ )
-  {
-    is_first_spike_ = false;
-    t_begin = 0;
-  }
-  else
+  if ( t_previous_spike_ != 0 )
```

Comment:
I think this new logic depends on  `t_previous_spike` being zero when `send` is called for the first time, otherwise it would lead to problems. I think we need to investigate what might happen is a new synapse is created using the copy or assignment constructor.  Looking at those functions is a TODO that I must add. 

In any case I think then an initialization as:

`long t_previous_spike_ = 0;`

might inform better about its intent better than

` , t_previous_spike_( 0 )`